ewcfg.py

```diff
@@ -1771,6 +1771,25 @@ def wef_tool(ctn = None):
 		ctn.crit = True
 		ctn.slimes_damage *= 2
 
+# weapon effect function for "bass"
+def wef_bass(ctn = None):
```

Comment:
the weapon effect is fine in principle, but meaty's weapon update is just around the corner, for which we changed all of the weapon effects. please talk to meaty about adjusting this effect for his update.

Reply:
It is just the bat weapon effect